sessions/js-forms-2/simple-live-calculator/README.md

```diff
@@ -0,0 +1,19 @@
+# Simple Live Calculator
+
+Let's build a simple live calculator!
+
+## Task
+
+There are two form fields where the user can enter a value.
+As soon as one of these values change, both values shall be multiplied. The result is to be rendered with the help of the output element.
+
+## Hints
+
+The following hints might guide you:
+
+- You may need to declare global `let` variables for both input fields and initialize them with a starting number.
+- Consider the `input` event for both input fields; on each input, you want to set the event target's value to your global variables.
+- If you calculate the result with a `calculateResult()` function, you can set the `textContent` of your output to this result.
+- Don't forget to call `calculateResult()` on every input event!
```

Comment:
These hints suggest a preferred way of solving the task, do we want that? I mean you can finish the task without such a function.

Reply:
I rewrote the hints, may you have a look again, @F-Kirchhoff?